kafka/manager.go

```diff
@@ -277,6 +277,9 @@ func (m *Manager) MonitorConsumerLag(topicConsumers []apmqueue.TopicConsumer) (m
 				}
 				for partition, lag := range partitions {
 					if lag.Err != nil {
+						if lag.Err == kerr.UnknownTopicOrPartition && !m.cfg.LogUnknownTopicWarning {
```

Comment:
To avoid adding another config, what do you think of using a debug log  for the specific `UnknownTopicOrPartition` error?
```go
if lag.Err == kerr.UnknownTopicOrPartition {
	logger.Debug(...)
)
} else {
   logger.Warn(...)
}
continue
```